composer.json

```diff
@@ -26,10 +26,10 @@
         }
     },
     "require-dev": {
-        "orchestra/testbench": "^3.6",
-        "orchestra/database": "^3.6",
+        "orchestra/testbench": "^3.8",
+        "orchestra/database": "^3.8",
         "mockery/mockery": "^1.0",
-        "phpunit/phpunit": "~7.0",
+        "phpunit/phpunit": "~8.0",
```

Comment:
Tests are broken because of PHPUnit 8.0

Reply:
With PHPUnit 7.0 they are even more broken. However in usage all seems fine.
Please see into this as it may only be some deeper dependency issue.

Reply:
Tests are now patched for PHPUnit 8.0 / PHP 7.2.